ctrlf.el

```diff
@@ -699,7 +704,10 @@ later (this should be used at the end of the search)."
             (if (and (not skip-search)
                      (ctrlf--search input :bound 'wraparound))
                 (progn
-                  (goto-char (match-beginning 0))
+                  (goto-char (or
+                              (and ctrlf-go-to-end-of-match
+                                   (match-end 0))
+                              (match-beginning 0)))
```

Comment:
Would it be better to use `if` instead of a combination of `or` and `and`?

Reply:
IDK probably? How do people handle such things? I don't have programming experience

Reply:
See https://github.com/raxod502/ctrlf/pull/73#discussion_r553104765 below.